alphabert/perception/world_building_task.py

```diff
@@ -0,0 +1,41 @@
+from enum import Enum
+
+
+class WorldBuildingTask(Enum):
```

Comment:
This class might not be useful for us at this point, in which case, I will be happy to remove it for now.